packages/e2e-tests/newUser.crossplatform.test.ts

```diff
@@ -50,7 +50,7 @@ describe('New User', () => {
     await buildSetup2.killChromeDriver()
   })
   describe('Stages:', () => {
-    it('Close debug modal', async () => {
+    it.skip('Close debug modal', async () => {
```

Comment:
Are those skipped in purpose?

Reply:
Yes, because now is tested new released version without this debug modal